frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -46,16 +49,27 @@ export default ({ account }: Props) => {
             text={<Balance amount={account.locked} />}
           />
         </Col>
-        <Col md="4">
+        <Col md="3">
           <CardCell
-            title="Storage (Used / Paid)"
+            title="Storage Used (Bytes)"
+            imgLink="/static/images/icon-storage.svg"
+            text={`${account.storageUsage.toLocaleString()} B`}
+          />
+        </Col>
+        <Col md="2">
+          <CardCell
+            title="Last Paid"
             imgLink="/static/images/icon-m-block.svg"
-            text={`${account.storageUsage.toLocaleString()} / ${account.storagePaidAt.toLocaleString()}`}
+            text={
+              <BlockLink blockHash={blockHash}>
+                {account.storagePaidAt}
```

Comment:
When we refer to block height, we use `#` in front of the number.

Reply:
added